components/main-menu.tsx

```diff
@@ -0,0 +1,55 @@
+import Image from "next/image"
+import Link from "next/link"
+import browserIcon from "@/assets/browser-icon.svg"
+import xIcon from "@/assets/x-icon.svg"
+
+interface MainMenuProps {
+  onClose: () => void
+}
+
```

Comment:
available in site config

Reply:
Fixed in: https://github.com/State-Channel-4/frontend/pull/27/commits/0e40b098777b2db2cbadddf0370f87d6795702c9